Source/JavaScriptCore/runtime/TemporalDuration.cpp

```diff
@@ -549,7 +553,9 @@ double TemporalDuration::total(JSGlobalObject* globalObject, JSValue optionsValu
     }
 
     ISO8601::Duration newDuration = m_duration;
-    balance(newDuration, unit);
+    auto infiniteResult = balance(newDuration, unit);
+    if (infiniteResult)
```

Comment:
I kind of prefer "infiniteResult.has_value()" in code like this. Otherwise it's too easy to read this as a check for zero. I'm not sure this is a project-wide consensus though.

Reply:
Hmm, seems like this could be one of those JSC vs. everybody else things. 😅 `.has_value())` (_with trailing paren_) has nearly no results in JSC, despite having a bunch in WebCore & WebKit. Since we've been using this style quite a bit in neighboring code, I'm inclined to leave it as-is, as long as you don't object.